.idea/shelf/Uncommitted_changes_before_Update_at_12_11_2022_3_23_PM_[Changes]/shelved.patch

```diff
@@ -0,0 +1,370 @@
+Index: Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java
```

Comment:
What is this file? My best guess is that this is some weird git stuff because the branch got wonky. If you don't need it delete it. If you need it you're probably in trouble since it's not being used in Android Studio.

Reply:
deleted